src/pool-bin/interfaces/IBinPoolManager.sol

```diff
@@ -73,13 +73,15 @@ interface IBinPoolManager is IProtocolFees, IPoolManager, IExtsload {
     /// @param id The abi encoded hash of the pool key struct for the pool that was modified
     /// @param sender The address that modified the pool
     /// @param ids List of binId with liquidity added
+    /// @param salt The salt to distinguish different mint from the same owner
     /// @param amounts List of amount added to each bin
     /// @param compositionFee fee occurred
     /// @param pFee Protocol fee from the swap: token0 and token1 amount
     event Mint(
         PoolId indexed id,
         address indexed sender,
         uint256[] ids,
+        bytes32 salt,
```

Comment:
just wondering if there will be cases requiring `bytes32[] salts`?

Reply:
I guess that might make Mint quite complicated 😂...  ppl could actually separate it into multiple calls if they want to do so